Automattic/jetpack
# Update/refactor number format location

#### Changes proposed in this Pull Request:

* This PR refactors the numberFormat component by moving it into the components JS package so it can be imported via `@automattic/jetpack-components`

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Apply this branch to your Jetpack testing environment
* If you are running a local environment, build `plugins/jetpack`
* If the build completes without errors and the Jetpack dashboard loads as expected, this change can be considered successful - there are no functional changes, only inclusion paths have changed.
* As an extra gut check, the numbers in the stats section on the Jetpack dashboard use the `numberFormat` function - these numbers should display as expected.
